src/app/Component/vote-poll/vote-poll.component.css

```diff
@@ -0,0 +1,15 @@
+button {
```

Comment:
can you use hierarchy of class rather than use a genric names like button, i, h3 etc